docs/output.md

```diff
@@ -224,6 +225,21 @@ Reporting:
 
 [mafft](https://github.com/GSLBiotech/mafft) is a fast but not very sensitive multiple sequence alignment tool.
 
+### untar
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `untar/`
```

Comment:
I would keep them compressed

Reply:
Hmm..Then the groovy function to check validity across input HMMs-MSAs would need to extract the filenames somehow anyway, right? And can CAT_CAT cat tar.gz files for the concatenated hmm.gz? Then also, can SeqKit extract ungapped fasta from tar.gz, for each family?
I think the complexity will rise quite a bit if we don't decompress.